src/oom/oomd-util.c

```diff
@@ -413,6 +413,24 @@ int oomd_insert_cgroup_context(Hashmap *old_h, Hashmap *new_h, const char *path)
         return 0;
 }
 
+void oomd_update_cgroup_contexts_between_hashmaps(Hashmap *old_h, Hashmap *curr_h) {
+        OomdCGroupContext *ctx;
+
+        assert(old_h);
+        assert(curr_h);
+
+        HASHMAP_FOREACH(ctx, curr_h) {
+                OomdCGroupContext *old_ctx;
+
+                old_ctx = hashmap_get(old_h, ctx->path);
+                if (old_ctx) {
```

Comment:
doesn't matter, but in the interest of keeping indentation levels short I'd reverse this check to be:

```c
if (!old_ctx)
        continue;
```

i.e. "early exit" is a bit nicer than "deep nesting" I thinkg